supplementary_style_guide/glossary_terms_conventions/general_conventions/t.adoc

```diff
@@ -56,6 +56,16 @@
 
 *See also*:
 
+[discrete]
+[[developer-preview]]
+==== image:images/yes.png[yes] Developer Preview (noun)
+*Description*: Developer Preview features provide early access to upcoming product features in advance of their possible inclusion in a Red Hat product offering, enabling customers to test functionality and provide feedback during the development process. These features might not have any documentation, are subject to change or removal at any time, and testing is limited.
```

Comment:
I would include that dev preview features are "no supported in any way by Red Hat" from the support scope statement.